app/page.tsx

```diff
@@ -42,20 +42,19 @@ export default function Home() {
     getUpcomingEvents()
```

Comment:
Use separate useEffect for countdown.
Current approach calls getUpcomingEvents every second, increasing the load on server.